composeApp/src/commonMain/kotlin/com/arkivanov/minesweeper/game/DefaultGameComponent.kt

```diff
@@ -24,6 +31,17 @@ internal class DefaultGameComponent(
 
     init {
         stateKeeper.register(key = KEY_SAVED_STATE, strategy = GameState.serializer()) { store.state }
+        scope.launch {
+            state
+                .map { it.gameStatus == GameStatus.STARTED }
+                .let { isStarted ->
```

Comment:
This is incorrect. Please pay attention to the details in the code snippet [here](https://github.com/arkivanov/Minesweeper/pull/15#issuecomment-1986922426). This code missing `distinctUntilChanged()` and `collectLatest {...}`.

Reply:
Your code snippet seems awesome! I'm stuck with an idea in your code to gather stateFlow from a store

Reply:
Yes, I think you should also use `store.stateFlow` instead of `state`.

Reply:
How to convert store.state to Flow? I'm doesn't see any extension functions at Decompose library for Value type.

Should I change type of a state (start using Flow, instead of Value for GameComponent.state?) or create an extension function that create flow from GameComponent.state?

Reply:
Oh, I think we should also add `com.arkivanov.mvikotlin:mvikotlin-extensions-coroutines:4.0.0-alpha02` to `commonMain`. After that there should be `store.stateFlow` extension available.

Reply:
to be more, understandable, this is what I got, when write your snippet
<img width="873" alt="image" src="https://github.com/arkivanov/Minesweeper/assets/55315908/98e5815d-a16a-4b69-9b1e-0df34f825641">

Reply:
> Oh, I think we should also add `com.arkivanov.mvikotlin:mvikotlin-extensions-coroutines:4.0.0-alpha02` to `commonMain`. After that there should be `store.stateFlow` extension available.

yea, its what I talking about 😅, I'm not so aware of all your libraries that you made, let me try it!

Reply:
> Oh, I think we should also add `com.arkivanov.mvikotlin:mvikotlin-extensions-coroutines:4.0.0-alpha02` to `commonMain`. After that there should be `store.stateFlow` extension available.

Very thanks for opportunity of fast replies, I appreciate it.

Reply:
> This is incorrect. Please pay attention to the details in the code snippet [here](https://github.com/arkivanov/Minesweeper/pull/15#issuecomment-1986922426). This code missing `distinctUntilChanged()` and `collectLatest {...}`.

Awesome cache-hit in a first try of this code snipper:)

Also get a some not bad record in a first try:D
<img width="458" alt="image" src="https://github.com/arkivanov/Minesweeper/assets/55315908/f7b25cb9-b20f-495d-a49c-a4b2e30e4a4a">

I'd like to think about what we can do more for this project? Have you thought about it?

Maybe local leaderboard, I think it is still one of classic ways of realisation.

Reply:
Make a commit with update, where fix stopwatch! Also wants write some tests!